### PR TITLE
Add small note about CPU/mem usage

### PR DESCRIPTION
**Pull request by rata**
My main concern to run this in my production k8s cluster was it's
CPU/mem usage. So, for example, if one node dies I was afraid this pod
might get scheduled and not others, more important, pods.

But, looking at it, it's really lightweight and can fit in any node just
fine without causing any issues (nor having more nodes). So, it may be
useful for other to see there is really no risk.

### CODE REVIEW

**Comment by philips** [comment]
Should we just default the resource to this?

**Comment by rata** [comment]
On Tue, Oct 04, 2016 at 07:34:56AM -0700, Brandon Philips wrote:

> Should we just default the resource to this?

Didn't look at the code to see if this is expected to remain true in bigger
clusters. Also, I don't know if we want to make such a promise now.  Maybe it's
too soon and we might want get something slightly more expensive (`1m` cpu is
really nothing).

But a note saying it's lightweight and an example of how small this can be for a
small cluster, I think it helps to make the reader not hesistate on this (or not
much).

When it's more mature, we may feel confident to do it, but I think now might be
too soon (although I'm not involved with the code, maybe it's not too soon!).
Or, when there is an update strategy, this might not even be needed :)

Thanks a lot,
Rodrigo

**Comment by rata** [comment]
@philips friendly ping? :)

**Comment by philips** [comment]
Sure, LGTM

File: README.md [hunk 1]
```diff
@@ -143,6 +143,13 @@ If you want to save the YAML that this produces, you can simply run:
 kubectl get deployment spartakus -o yaml
 ```
 
+Also, you shouldn't worry about CPU and mem usage of this pod, as it is very
+lightweight. You can edit the deployment to request only a small amount of CPU
+and memory and make sure this won't consume more. For example, this will work
+totally fine with `1m` CPU and `10Mi` mem on a 5 nodes cluster.
+
+So, don't be afraid about it's CPU/mem usage :)
```
Comment (squat) [inline README.md]: Nitpick: please change to `its`

**Comment by rata** [comment]
@squat fixed, thanks!

**Comment by thockin** [comment]
I am OK with this.  I would be OK changing the default to 1m CPU request and 10m limit.  Memory request just needs to be enough to not OOM.

**Pull request merged**